java-operators/src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+
```

Comment:
I think we could remove this file. There's no need of properties for this topic.

Reply:
done